homeassistant/components/webostv/media_player.py

```diff
@@ -450,6 +451,9 @@ async def async_button(self, button):
         await self._client.button(button)
 
     @cmd
-    async def async_command(self, command):
+    async def async_command(self, command, **kwargs):
         """Send a command."""
-        await self._client.request(command)
+        argument = kwargs.get("argument")
+        if argument is not None:
+            argument = json.loads(argument)
```

Comment:
What is argument is not a JSON ?

Reply:
oh yes that should be handled, I'll add that in

Reply:
i switched it to use a dict instead of json string so we can avoid this issue